doc/manuals/user/append_and_delete.md

```diff
@@ -7,185 +7,180 @@ with an example.
-      EOF                                                                                                                       
+     (curl localhost:1026/v1/updateContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+     {
+      "contextElements": [
+      {
+         "type": "T",
+         "isPattern": "false",
+         "id": "E1",
+         "attributes": [
+         {
+              "name": "A",
+              "type": "TA",
+              "value": ""
+         }
+         ]
+       }
+       ],
+        "updateAction": "DELETE"
+     }
+     EOF                                                                                                                      
   
 Now, a query to the entity shows attribute B:
 
-      curl localhost:1026/v1/contextEntities/E1 -s -S --header 'Content-Type: application/xml' | xmllint --format -       curl localhost:1026/v1/contextEntities/E1 -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' | python -mjson.tool
-
-      <?xml version="1.0"?>                          {
-      <contextElementResponse>                           "contextElement": {
-        <contextElement>                                     "attributes": [
-          <entityId type="" isPattern="false">                   {
-            <id>E1</id>                                              "name": "B",
-          </entityId>                                                "type": "TB",
-          <contextAttributeList>                                     "value": "2"
-            <contextAttribute>                                   }
-              <name>B</name>                                 ],
-              <type>TB</type>                                "id": "E1",
-              <contextValue>2</contextValue>                 "isPattern": "false",
-            </contextAttribute>                              "type": ""
-          </contextAttributeList>                        },
-        </contextElement>                                "statusCode": {
-        <statusCode>                                         "code": "200",
-          <code>200</code>                                   "reasonPhrase": "OK"
-          <reasonPhrase>OK</reasonPhrase>                }
-        </statusCode>                                }
-      </contextElementResponse>                  
+      curl localhost:1026/v1/contextEntities/E1 -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' | python -mjson.tool
+
+      {
+      "contextElement": {
+        "attributes": [
+        {
+            "name": "B",
+            "type": "TB",
+            "value": "2"
+        }
+        ],
+            "id": "E1",
+            "isPattern": "false",
+            "type": ""
+        },
+        "statusCode": {
+	  "code": "200",
+          "reasonPhrase": "OK"
+          }
```

Comment:
Yes.. check the rendering at https://github.com/telefonicaid/fiware-orion/blob/hardening/RemovedXML/doc/manuals/user/append_and_delete.md. It looks with a weird indentation.